vncviewer/vncviewer.cxx

```diff
@@ -678,9 +717,8 @@ int main(int argc, char** argv)
           }
       }
     } catch (rdr::Exception& e) {
-      vlog.error("%s", e.str());
```

Comment:
Did you have a specific plan when moving these?

Reply:
abort_vncviewer is already logging the error so no need to do it twice I thought.

Reply:
The message given to `abort_vncviewer()` is something we show to the user, which isn't always the same thing we want to put in a log. So I think it is best to separate the two, just like we had before.

Reply:
OK fine, it just looked inconsistent to me that sometimes something is logged and in other cases not and thought the easiest is to have the logging at a central place.